doc/sphinx/source/framework-integration-guides.rst

```diff
@@ -46,55 +46,31 @@ system that already has an ``ngraph_dist`` installed.
    .. code-block:: console
 
       $ sudo apt-get install python3-pip python3-venv
+      $ python3 -m venv .venv
+      (.venv)$ . bin/activate
 
+#. Go to the "python" subdirectory of the ``ngraph`` repo and complete the 
+   steps needed to get the wrapper installed: 
 
-#. Clone the ngraph-neon integration repo somewhere you can find it easily. The
-   install configuration we'll document here places it under a directory we'll 
-   create named ``frameworks``
-
-   * This section is optional, but it is the configuration we'll assume for the 
-     ``ngraph-neon`` documenation:
-
-      .. code-block:: console
+   .. code-block:: console
 
-         $ sudo mkdir -p /opt/frameworks   
-         $ sudo chown -R username:username /opt/frameworks
-         $ cd /opt/frameworks
+      (.venv)$ cd ngraph/python
+      (.venv)$ git clone --recursive -b allow-nonconstructible-holders https://github.com/jagerman/pybind11.git
+      (.venv)$ pip install -U . 
 
-   * This section is required:       
+#. Finally are we ready to install the `neon` integration: 
 
-      .. code-block:: console
+   .. code-block:: console
 
-         $ git clone git@github.com:NervanaSystems/ngraph-neon
-         $ cd ngraph-neon
+      (.venv)$ git clone git@github.com:NervanaSystems/ngraph-neon
+      (.venv)$ cd ngraph-neon
+      (.venv)$ make install
 
-#. *Optional* Activate a virtualenv if you prefer working with virtualenvs:
+#. To test a training example, you can run the following from ``ngraph-neon/examples/cifar10``
```

Comment:
instead of words here ..how about giving the command with the entire path in the next step...
python examples/cifar10/cifar10_conv.py

Reply:
@postrational had commented that it's not clear which directory holds the correct "example" to run whether it's the ngraph-python repo or the ngraph-neon we cloned.  Change is not needed.

Reply:
ok. i will get rid of examples from ngraph/python just to avoid confusion. They were added there just for testing the wrapper code.

Reply:
I think that we can be more clear in the docs that an example comes from this or that repo. We may want to keep `examples` in the ngraph repo.